### ISSUE TITLE
Error dependency failed to start: container sentry-self-hosted-web-1 is unhealthy

### ISSUE THREAD

**Issue by ghun131** [getsentry/self-hosted#2562]
### Self-Hosted Version

23.10.1

### CPU Architecture

x86_64

### Docker Version

24.0.7

### Docker Compose Version

2.21.0

### Steps to Reproduce

1. Download the package here https://github.com/getsentry/self-hosted/releases/tag/23.10.1
2. Unzip and run `./install.sh`
3. When installation was successful, run `docker compose up -d`
4. See the error and `http://127.0.0.1:9000/` was inaccessible 

### Expected Result

Can access http://127.0.0.1:9000/ and `docker compose up -d` is working

### Actual Result

- docker compose log is here https://gist.github.com/ghun131/024a20726e0e03feb22b8564423aa717
- installation log is here https://gist.github.com/ghun131/6808015df5b4ad62ebd8c5f6d2fe5096

**Comment by hubertdeng123** [comment]
Seeing `Error: Configuration file does not exist. Use 'sentry init' to initialize the file.` in your logs. I would want to make sure a `sentry.conf.py` exists in your installation. Weird though, because that should be created when you ran the `./install.sh` script.

**Comment by ghun131** [comment]
I have no idea but I gave up on installing on my Debian machine. Instead I hire an instance on AWS LightSail using Ubuntu. I successfully installed and run all the containers. Maybe it is about the number of CPUs and RAM. My local machine only have 2 CPUs and I encountered similar issue with 2CPUs instance on LightSail. But everything works well with a 4CPUs + 8GB instance. I'm closing this issue. Thank you for the reply
